proto/target.proto

```diff
@@ -137,3 +137,41 @@ message GetTargetResponse {
   // The pagination token to retrieve the next page of results.
   string next_page_token = 4;
 }
+
+message Artifact {
+  string name = 1;
+  string bytestream_url = 2;
+}
+
+message Target {
```

Comment:
think we might also need actionExecuted here (stderr etc for failed tests)

Reply:
i think we could just leave this in the GetInvocation response the way we get it now

it's hard to match to a target, and according to the BES proto we'll only get "important ones". i've only ever seen one in a response